src/modules/bordList.js

```diff
@@ -1,26 +1,22 @@
 const bordList = () => {
+  const scores = JSON.parse(localStorage.getItem('scores')) || [];
   const board = document.getElementById('board');
+  const data = scores.map((s) => '<tr><td>'.concat(s.user).concat(' : ').concat(s.score).concat('</td></tr>'));
   const list = `
         <div>
-            <h3>Recent Scores <button>Refresh</button></h3>
+            <h3>Recent Scores <button id="refresh">Refresh</button></h3>
```

Comment:
I suspect this might be the reason why your `Refresh` button doesn't work sometimes, the issue being that when the `eventListener` is added, this list hasn't been generated, therefore the `eventListener` is added on a nonexistent element ❌ in order to fix that you can either use the `DOMSubtreeModified` event to add the listener when the list is added and not before. What is that you may be wondering? 🤔 It fires on a node when a modification occurs in the subtree that belongs to it. The `DOMSubtreeModified` event can be used instead of the other mutation events (`DOMAttrModified`, `DOMCharacterDataModified`, `DOMNodeInserted`, `DOMNodeInsertedIntoDocument`, `DOMNodeRemoved` and `DOMNodeRemovedFromDocument`). 🤓 

The second option, is much more simple, since the `Refresh` button is static and doesn't change based on the list, you could consider having it directly in the `HTML` template, I don't think it's required to be generated from the `JS`, if it's you should ignore this second option, so should the next reviewer. ⚠️